config/application.yml.default

```diff
@@ -318,7 +318,9 @@ sp_issuer_user_counts_report_configs: '[]'
 team_ada_email: ''
 team_all_login_emails: '[]'
 team_daily_reports_emails: '[]'
+team_fraud_reports_emails: '[]'
 team_ursula_email: ''
+test_fraud_reports_emails: '[]'
```

Comment:
I'd call it `team_daily_fraud_reports_emails` since they're not exactly tests
```suggestion
team_daily_fraud_reports_emails: '[]'
```

Reply:
We changed this to team_daily_fraud_metrics_emails, as team Judy will be making their own 'reports' from this data